packages/client-core/src/admin/components/Project/GithubRepoDrawer.tsx

```diff
@@ -44,7 +44,6 @@ const GithubRepoDrawer = ({ open, project, onClose }: Props) => {
       setProcessing(false)
       onClose()
     } catch (err) {
-      console.error(err)
```

Comment:
Since we're doing `NotificationService.dispatchNotify()`, it will log the error, no need to do it here.